--- a/CMakeLists.txt 	2024-11-08 10:33:32.649036467 +0200
+++ b/CMakeLists.txt 	2024-11-08 10:33:39.855560407 +0200
Upstream-Status: Inappropriate [local build fix]
@@ -5,7 +5,7 @@
 add_definitions(-O -Wall -Werror --std=gnu99 -D_GNU_SOURCE)
 
 add_library(vuci MODULE vuci.c)
-target_link_libraries(vuci crypt;ubus)
+target_link_libraries(vuci ubus)
 set_target_properties(vuci PROPERTIES OUTPUT_NAME vuci PREFIX "")
 
 install(TARGETS vuci
