--- a/Makefile	2024-11-08 13:44:06.976494666 +0200
+++ b/Makefile	2024-11-08 13:44:21.542860764 +0200
@@ -11,7 +11,7 @@
 LIBNL_OBJ=lua_crypto.o
 
 $(LIBNAME): $(LIBNL_OBJ)
-	$(CC) -Wall -lssl -lcrypto -llua5.1 -shared -fPIC -DLUA_USE_POSIX -DLUA_USE_DLOPEN -o $@ $^ -I/usr/include/lua5.1
+	$(CC) -Wall -lssl -lcrypto -llua5.1 -shared -fPIC -DLUA_USE_POSIX -DLUA_USE_DLOPEN $(LDFLAGS) -o $@ $^ -I/usr/include/lua5.1
 
 clean:
 	rm -rf *.o *.so
